### PR TITLE
Ego refactor (All EGO PRs will be frozen until this is merged.)

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Refactors EGO into separate sprite files, and item paths.

## Why It's Good For The Game
So, Byond has a limit on how many item states we can have in a file; and we're going to be pushing that 1000 cap soon

## Changelog
:cl:
refactor: refactored EGO sprites and code.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/ego_gear/he.dm [hunk 1]
```diff
@@ -166,33 +174,33 @@
 								FORTITUDE_ATTRIBUTE = 40
 								)
 
-/obj/item/clothing/suit/armor/ego_gear/brick
+/obj/item/clothing/suit/armor/ego_gear/he/brick
 	name = "yellow brick"
 	desc = "The heavy dress weighs you down, smacking against your knees."
 	icon_state = "brick"
-	armor = list(RED_DAMAGE = -20, WHITE_DAMAGE = 50, BLACK_DAMAGE = 30, PALE_DAMAGE = 10)
+	armor = list(RED_DAMAGE = 30, WHITE_DAMAGE = 50, BLACK_DAMAGE = -20, PALE_DAMAGE = 10)
```
Comment (Gboster-0) [inline code/modules/clothing/suits/ego_gear/he.dm:181]: ya should probably mention this change, if its intentional

**Review by EgorDinamit — APPROVED** [review]
This is good, I guess. Can't believe we're approaching a limit.

File: code/modules/clothing/suits/ego_gear/he.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/modules/clothing/suits/ego_gear/he.dm:181]: Uh oh, random EGO rebalance??

Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/he.dm:181]: Unused armor, same numbers as another EGO

**Pull request merged**